nbbuild/cluster.properties

```diff
@@ -717,8 +717,6 @@ nb.cluster.java=\
         maven.search,\
         maven.spring,\
         nashorn.execution,\
-        performance,\
```

Comment:
This is somewhat controversial; @jlahoda or @jtulach to review: performance tests are not really used now, and the performance* modules contain just testing code, yet they appear in the release distribution. And fail commit-validation tests.

Reply:
This change probably results in [following failure](https://travis-ci.org/github/apache/netbeans/jobs/728712951):
```
[nb-javac] /home/travis/build/apache/netbeans/enterprise/performance.javaee/test/unit/src/org/netbeans/performance/j2ee/startup/MeasureJ2EEStartupTime.java:23: error: package org.netbeans.modules.performance.utilities does not exist

 [nb-javac] import org.netbeans.modules.performance.utilities.MeasureStartupTimeTestCase;

 [nb-javac]                                                  ^

 [nb-javac] /home/travis/build/apache/netbeans/enterprise/performance.javaee/test/unit/src/org/netbeans/performance/j2ee/startup/MeasureJ2EEStartupTime.java:30: error: cannot find symbol

 [nb-javac] public class MeasureJ2EEStartupTime extends MeasureStartupTimeTestCase {
```

Reply:
ah ! Strange I didn't get this one running build. So perhaps remove `performance.javaee` as well ?